chia/full_node/mempool.py

```diff
@@ -47,6 +47,11 @@
 # integers, which we rely on for computing fee per cost as well as the fee sum
 MEMPOOL_ITEM_FEE_LIMIT = 2**50
 
+# Block overhead cost calculation
+QUOTE_BYTES = 2
+QUOTE_EXECUTION_COST = 20
+BLOCK_OVERHEAD = QUOTE_BYTES * DEFAULT_CONSTANTS.COST_PER_BYTE + QUOTE_EXECUTION_COST
```

Comment:
I think it would be better to use the `ConsensusConstants` we pass around and instantiate the node with, rather than assuming they are the defaults. Does the mempool not have access to the `ConsensusConstants`?